src/Columns/ColumnAggregateFunction.cpp

```diff
@@ -56,7 +63,7 @@ static String getTypeString(const AggregateFunctionPtr & func)
 
 
 ColumnAggregateFunction::ColumnAggregateFunction(const AggregateFunctionPtr & func_, std::optional<size_t> version_)
-    : func(func_), type_string(getTypeString(func)), version(version_)
+    : func(func_), type_string(getTypeString(func, version_)), version(version_)
```

Comment:
Also may be in line 79.. Passing version to `set` from here
https://github.com/ClickHouse/ClickHouse/blob/ae214ffc5b6c74021666f9f0de3fda7c71acde85/src/DataTypes/Serializations/SerializationAggregateFunction.cpp#L82 (there is version field in class too)
As I see from a quick look it is the only place where set is used.